chapters/functions.tex

```diff
@@ -2252,11 +2252,11 @@ \subsection{Annotations for External Functions}\label{annotations-for-external-l
   \end{nonnormative}
 \item
   The
-  {\lstinline!annotation(IncludeDirectory="modelica:/ModelicaLibraryName/Resources/Include")!}\annotationindex{IncludeDirectory}, used to specify a location for header files.
+  {\lstinline!annotation(IncludeDirectory="modelica:/PackageName/Resources/Include")!}\annotationindex{IncludeDirectory}, used to specify a location for header files.
   The preceding one is the default and need not be specified; but another location could be specified by using an URI name for the include directory, see \cref{external-resources}.
```

Comment:
I realize that something important is being lost regarding the default; it needs to be clear that the default is the name of the current library's top-level package, and `PackageName` isn't making this clear.

Something like the following would preserve the meaning and still drastically shorten the `\lstinline` arguments:
```suggestion
  {\lstinline!annotation(IncludeDirectory=$\mathit{uri}$)!}\annotationindex{IncludeDirectory}, used to specify a URI location (\cref{external-resources}) for header files.
  The default $uri$ is \lstinline!"modelica:/ModelicaLibraryName/Resources/Include"! and need not be specified.
```

Reply:
There is an explanation for PackageName (changed from the previous name in this PR) after the list.

Obviously we can improve that, but to me that would be a separate PR, as this was just intended to have shorter names to fix two issues.

Reply:
Separate PR if needed.